src/walletConnect/screens/ActionRequest.test.tsx

```diff
@@ -242,12 +268,7 @@ describe('ActionRequest with WalletConnect V2', () => {
         )
       ).toBeTruthy()
       expect(getByText('dappsDisclaimerUnlistedDapp')).toBeTruthy()
-      expect(
-        getByText('walletConnectRequest.estimatedNetworkFee, {"networkName":"Celo Alfajores"}')
-      ).toBeTruthy()
-      const fee = within(getByTestId('EstimatedNetworkFee'))
-      expect(fee.getByText('0.0012 CELO')).toBeTruthy()
-      expect(fee.getByText('₱0.008')).toBeTruthy()
+      expect(queryByTestId('EstimatedNetworkFee')).toBeFalsy()
```

Comment:
This was accidentally taking the viem enabled path.
I made sure to reset the feature flag mock before each test to avoid this and moved this check to where it belonged.
This test case was testing `personal_sign` which shouldn't have a TX fee anyway.